### ISSUE TITLE
When setting a value in server-side `ValueChangeEvent.isFromClient()` should be false

### ISSUE THREAD

**Issue by manolo** [vaadin/vaadin-combo-box-flow#53]
In `ComboBox` `ValueChangeEvent` does not bring the `isFromClient` flag set to false when the value is set in server side. Otherwise, the same code in `DatePicker` or `TextField` works alright. 

Run this code and you can see that only in combo-box the flag is wrong

```
public TestView() {
  TextField tf = new TextField();
  tf.addValueChangeListener(
      e -> System.err.println(e.isFromClient() + " " + e.getOldValue() + " -> " + e.getValue()));

  DatePicker dt = new DatePicker();
  dt.addValueChangeListener(
      e -> System.err.println(e.isFromClient() + " " + e.getOldValue() + " -> " + e.getValue()));

  ComboBox<String> cb = new ComboBox<>();
  cb.setDataProvider(new AbstractBackEndDataProvider<String, String>() {
    protected Stream<String> fetchFromBackEnd(Query<String, String> query) {
      return Arrays.asList("Foo", "Bar").stream();
    }
    protected int sizeInBackEnd(Query<String, String> query) {
      return 2;
    }
  });
  cb.addValueChangeListener(
      e -> System.err.println(e.isFromClient() + " " + e.getOldValue() + " -> " + e.getValue()));

  add(cb, dt, tf);

  tf.setValue("Baz");
  dt.setValue(LocalDate.now());
  cb.setValue("Foo");
}
```

The output is

```
false  -> Baz
false null -> 2018-02-19
true null -> Foo
```
But should be
```
false  -> Baz
false null -> 2018-02-19
false null -> Foo
```

**Comment by Legioth** [comment]
Most likely the same root cause as https://github.com/vaadin/vaadin-combo-box-flow/issues/48, i.e. that server-side `setValue` is hooked up to go through the client.

**Comment by denis-anisimov** [comment]
Most likely yes.
Should be checked once #48 is fixed (PR is already there, will be merged shortly).

**Comment by manolo** [comment]
Can someone set the `app starter` label here? I have not permissions.

**Comment by denis-anisimov** [comment]
Done.
But may be it's already fixed (though you need a snapshot) to check

**Comment by manolo** [comment]
Snapshot works with my demo project, and apparently with bakery, let's leave this opened until there is a platform release including this fix for bakery.

**Comment by manolo** [comment]
Is this fix going to be released with next platform release?

**Comment by denis-anisimov** [comment]
This ticket is the duplicate of #48.

The PR for this ticket is just about IT test for this usecase.

Anyway both are merged and should be released in the nearest upcoming release (today 22.02.2018).

**Comment by denis-anisimov** [comment]
I'm talking about Flow and flow components release though.
I don't know when new platform will be released.
